src/main/webapp/video.js

```diff
@@ -45,3 +43,31 @@ function onYouTubeIframeAPIReady() {
 function onPlayerReady(event) {
   event.target.playVideo();
 }
+
+/** Starts timer if {@code currState} is playing, stops otherwise. */
+function startStopTimerOnStateChange(currState) {
```

Comment:
for this method you should just pass state as that is all you care about.

Reply:
The api determines what to pass to this function. In this case they send an object with `.data` having the video state

Reply:
sgtm